+nosnoc/+solver/RelaxationSolver.m

```diff
@@ -54,34 +60,36 @@
                 % TODO(@anton) implement this
             else % Otherwise use vdx internally anyway but be sad about interleaving
                 use_vdx = false;
+                % preprocess empty fields
+                
                 nlp = vdx.Problem('casadi_type', casadi_symbolic_mode);
-                nlp.w.mpcc_w(0) = {mpcc.x};
-                nlp.p.mpcc_p(0) = {mpcc.p};
-                nlp.g.mpcc_g(0) = {mpcc.g};
-                nlp.p.sigma_p(0) = {{'sigma_p', 1}, 0, inf, opts.sigma_0};
+                nlp.w.mpcc_w = {mpcc.x};
+                nlp.p.mpcc_p = {mpcc.p};
+                nlp.g.mpcc_g = {mpcc.g};
+                nlp.p.sigma_p = {{'sigma_p', 1}, opts.sigma_0};
```

Comment:
can you please comment that an nlp is created out of the mpec symbolic functions.
nlp  - is a vdx object, nlp.w - is a variablevectors (or primal vectors, whatever is the official name), 
mpcc_w  is a variable group (right name?) that contains all variables of the initial mpcc. New vars might be added in the nlp due to the reformulation. 
(these comments will help one to get into reading vdx code faster, after some sytntax was forgotten)

Reply:
Comment added.